membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -29,28 +29,6 @@ class AttributeController extends Controller with LazyLogging {
   lazy val authenticationService: AuthenticationService = IdentityAuthService
   lazy val metrics = Metrics("AttributesController")
 
-  def update = BackendFromCookieAction.async { implicit request =>
```

Comment:
this was not used by anyone according to the fastly logs from 11th Jun 00:00 until 12th Jun 10:19am